src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Implementation.cs

```diff
@@ -1368,18 +942,17 @@ private async ValueTask<int> ReadAsyncInternal<TReadAdapter>(TReadAdapter adapte
                         {
                             if (!_sslAuthenticationOptions.AllowRenegotiation)
                             {
+                                if (NetEventSource.IsEnabled) NetEventSource.Fail(this, "Renegotiation request but it is disabled");
```

Comment:
"Renegotiation request but it is disabled" -> "Renegotiation was requested but it is disallowed"